src/q6/prostoch-INMA1731/exam/2018/Août/Majeure/prostoch-INMA1731-exam-2018-Août-Majeure.tex

```diff
@@ -0,0 +1,25 @@
+\documentclass[fr]{../../../../../../eplexam}
+
+\hypertitle{Processus Stochastiques}{6}{INMA}{1731}{2018}{Août}{Majeure}
+{Nicolas Dourt}
+{Pierre-Antoine Absil \and Luc Vandendorpe}
```

Comment:
Je pense que le `\and` ne donnera pas très bien...

Reply:
Oups ^^ ça m'apprendra.